client/my-sites/subscribers/components/subscriber-data-views/subscriber-data-views.tsx

```diff
@@ -7,24 +7,24 @@ import TimeSince from 'calypso/components/time-since';
 import { EmptyListView } from 'calypso/my-sites/subscribers/components/empty-list-view';
 import { SubscriberLaunchpad } from 'calypso/my-sites/subscribers/components/subscriber-launchpad';
 import { useSubscribersPage } from 'calypso/my-sites/subscribers/components/subscribers-page/subscribers-page-context';
-import { useSubscriptionPlans } from 'calypso/my-sites/subscribers/hooks';
+import { useSubscriptionPlans, useUnsubscribeModal } from 'calypso/my-sites/subscribers/hooks';
 import { Subscriber } from 'calypso/my-sites/subscribers/types';
 import { useSelector } from 'calypso/state';
+import { getCouponsAndGiftsEnabledForSiteId } from 'calypso/state/memberships/settings/selectors';
 import isAtomicSite from 'calypso/state/selectors/is-site-automated-transfer';
 import { isSimpleSite } from 'calypso/state/sites/selectors';
 import { SubscribersSortBy } from '../../constants';
 import { SubscriberDetails } from '../subscriber-details';
 import { SubscribersHeader } from '../subscribers-header';
+import { UnsubscribeModal } from '../unsubscribe-modal';
 import type { View, Action } from '@wordpress/dataviews';
 import './style.scss';
 
 type SubscriberDataViewsProps = {
 	siteId: number | undefined;
-	onClickView: ( subscriber: Subscriber ) => void;
```

Comment:
We don't need `onClickView` anymore, since we're selecting the subscriber instead.